pyproject.toml

```diff
@@ -25,7 +25,7 @@ classifiers = [
 "Discord" = "https://discord.gg/8WkYz3fNFm"
 
 [tool.poetry.dependencies]
-python = ">=3.8"
+python = ">=3.8,<3.11"
```

Comment:
Breaking changes on python 3.11?

Reply:
I don't think there are breaking changes for us.

Reply:
Why changing this then?

Reply:
@Sigmanificient @Arthurdw Poetry is very picky about supported version of dependencies. One such dependency (Nox I believe) does not support anything greater than 3.11 so the install fails. Unless this is added.